contentcuration/contentcuration/frontend/accounts/pages/Create.vue

```diff
@@ -2,35 +2,428 @@
+          :error-messages="emailErrors"
+          @input="emailErrors = []"
+        />
+        <PasswordField
+          v-model="form.password1"
+          :label="$tr('passwordLabel')"
+        />
+        <PasswordField
+          v-model="form.password2"
+          :additionalRules="passwordConfirmRules"
+          :label="$tr('confirmPasswordLabel')"
+        />
+
+        <!-- Usage -->
+        <VInput required :rules="usageRules" class="mt-2" />
+        <h1 class="font-weight-bold subheading mb-2">
+          {{ $tr('usageLabel') }}*
+        </h1>
+        <div v-for="option in usageOptions" :key="option.id">
+          <VCheckbox
+            v-model="form.uses"
+            :label="option.label"
+            :value="option.id"
+            hide-details
+            color="primary"
+          />
+          <TextField
+            v-if="showStorageField(option.id)"
+            v-model="form.storage"
+            :label="$tr('storingUsagePlaceholder')"
+            :placeholder="$tr('storingUsageExample')"
+            class="ml-4 my-1"
+          />
+          <TextArea
+            v-else-if="showOtherField(option.id)"
+            v-model="form.other_use"
+            :label="$tr('otherUsagePlaceholder')"
+            class="ml-4 my-2"
+          />
+        </div>
+
+
+        <!-- Location -->
+        <VInput required :rules="locationRules" class="mt-4" />
+        <h1 class="font-weight-bold subheading my-2">
+          {{ $tr('locationLabel') }}*
+        </h1>
+        <CountryField v-model="form.locations" />
+
+        <!-- Source -->
+        <VInput required :rules="sourceRules" class="mt-2" />
+        <h1 class="font-weight-bold subheading my-2">
+          {{ $tr('sourceLabel') }}*
+        </h1>
+        <VSelect
+          v-model="form.source"
+          :items="sourceOptions"
+          item-text="label"
+          item-value="id"
+          outline
+          menu-props="offsetY"
+          :label="$tr('sourcePlaceholder')"
+        />
+        <TextArea
+          v-if="form.source === sources.ORGANIZATION"
+          v-model="form.organization"
+          :label="$tr('organizationSourcePlaceholder')"
+        />
+        <TextArea
+          v-else-if="form.source === sources.CONFERENCE"
+          v-model="form.conference"
+          :label="$tr('conferenceSourcePlaceholder')"
+        />
+        <TextArea
+          v-else-if="form.source === sources.OTHER"
+          v-model="form.other_source"
+          :label="$tr('otherSourcePlaceholder')"
+        />
+
+        <!-- Policy -->
+        <ActionLink
+          class="mt-4"
+          :text="$tr('viewPrivacyPolicyLink')"
+          @click="showPolicies = true"
+        />
+        <VDialog v-model="showPolicies" width="500">
+          <VCard>
+            <VCardTitle class="headline grey lighten-4" primary-title>
+              {{ $tr('privacyPolicyTitle') }}
+            </VCardTitle>
+            <iframe :src="policyLink"></iframe>
```

Comment:
All of the policy text exists in the [backend templates folder](https://github.com/learningequality/studio/tree/develop/contentcuration/contentcuration/templates/policies/text). Because we have a view decorator, I didn't want to change too much on how we handle [policy detection](https://github.com/learningequality/studio/blob/develop/contentcuration/contentcuration/utils/policies.py) and maintaining a historical record of all the policies, so I just embedded an iframe for now until we hash it out later

Reply:
might be worth adding this comment as an actual code comment